src/Markdig/Renderers/Roundtrip/RoundtripRenderer.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Alexandre Mutel. All rights reserved.
+// This file is licensed under the BSD-Clause 2 license. 
+// See the license.txt file in the project root for more information.
+
+using System.IO;
+using Markdig.Syntax;
+using Markdig.Renderers.Roundtrip.Inlines;
+using Markdig.Helpers;
+
+namespace Markdig.Renderers.Roundtrip
+{
+    /// <summary>
+    /// Default HTML renderer for a Markdown <see cref="MarkdownDocument"/> object.
+    /// </summary>
+    public class RoundtripRenderer : TextRendererBase<RoundtripRenderer>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NormalizeRenderer"/> class.
```

Comment:
Default comments copied over, should update with more info for the new renderer.